FV3/io/FV3GFS_io.F90

```diff
@@ -3744,6 +3746,55 @@ subroutine store_data3D_coarse_pressure_level(id, name, method, nx, ny, nz, full
     endif
     used = send_data(id, coarse, Time)
 end subroutine store_data3D_coarse_pressure_level
+
+subroutine sfc_data_override(Time, IPD_data, Atm_block, Model)
+  ! Ported from the GFDL SHiELD_physics repository.
+  ! https://github.com/NOAA-GFDL/SHiELD_physics/blob/main/FV3GFS/FV3GFS_io.F90
+  implicit none
+  type(time_type), intent(in) :: Time
+  type(IPD_data_type), intent(inout) :: IPD_data(:)
+  type(block_control_type), intent(in) :: Atm_block
+  type(IPD_control_type), intent(in) :: Model
+
+  integer :: i, j, ix, nb
+  integer :: isc, jsc, iec, jec
+  logical :: used
+  real, allocatable :: sst(:,:), ci(:,:)
+
+  isc = Atm_block%isc
+  iec = Atm_block%iec
+  jsc = Atm_block%jsc
+  jec = Atm_block%jec
+
+  if (Model%use_prescribed_sst) then
+    ! Here is a sample data_table that will enable reading in
+    ! external SSTs and sea-ice from an external file.
+    !
+    !"ATM", "sst", "sst", "INPUT/ec_sst.nc", "bilinear", 1.0
+    !"ATM", "ci", "ci", "INPUT/ec_sst.nc", "bilinear", 1.0
+    allocate(sst(isc:iec,jsc:jec))
+    allocate(ci(isc:iec,jsc:jec))
+    call data_override('ATM', 'sst', sst, Time, override=used)
+    if (.not. used) then
+      call mpp_error(FATAL, " SST dataset not specified in data_table.")
```

Comment:
Yes, that is correct (though I think since a separate error is raised if the file cannot be found, this mainly checks if the `sst` variable is accounted for in the `data_table`).